ZAPD/Main.cpp

```diff
@@ -53,7 +50,7 @@ void ErrorHandler(int sig)
 	fprintf(stderr, "Traceback:\n");
 	for (size_t i = 1; i < size; i++)
 	{
-		Dl_info info;
+		Dl_info info;	
```

Comment:
Here is a weird extra space. Don't forget to run the formatter.
```suggestion
		Dl_info info;
```